crates/polars-plan/src/dsl/mod.rs

```diff
@@ -1308,6 +1308,21 @@ impl Expr {
         self.apply_private(FunctionExpr::Rank { options, seed })
     }
 
+    #[cfg(feature = "replace")]
+    /// Replace the given values with other values.
+    pub fn replace<E: Into<Expr>>(self, old: E, new: E, default: Option<E>) -> Expr {
+        match default {
+            Some(default) => {
```

Comment:
default is an unstable keyword I believe.

Reply:
Indeed, it's mentioned here:
https://rust-lang.github.io/rfcs/1210-impl-specialization.html

However, everything seems to compile fine even using `default` as a param/variable name 🤷‍♂️ 

I can rename everything to `default_` if you prefer?